Sources/ProjectSpec/Project.swift

```diff
@@ -25,7 +25,9 @@ public struct Project: BuildSettingsContainer {
     public var fileGroups: [String]
     public var configFiles: [String: String]
     public var include: [String] = []
+    
     private var targetsMap: [String: Target]
+    private var aggregateTargetsMap: [String: AggregateTarget]
```

Comment:
One missing map creation in the `aggregateTargets` setter

Reply:
Oops. Fixed.